packages/block-editor/src/components/block-list/block.js

```diff
@@ -305,6 +306,14 @@ const applyWithDispatch = withDispatch( ( dispatch, ownProps, { select } ) => {
 					getPreviousBlockClientId( clientId );
 				if ( previousBlockClientId ) {
 					mergeBlocks( previousBlockClientId, clientId );
+				} else {
+					const replacement = switchToBlockType(
```

Comment:
Sweet, but can we add a comment explaining what this is doing and what our intention is? The wildcard selector is not immediately obvious.

Reply:
Done